dockerExtension.ts

```diff
@@ -281,6 +298,75 @@ namespace Configuration {
     }
 }
 
+function registerDebugConfigurationProvider(ctx: vscode.ExtensionContext): void {

```

Comment:
Can you move this function into it's own file in the 'debugging' folder? We try to keep the main extension.ts file as clean as possible otherwise it gets out of hand quickly

Reply:
Sure! Moved.